index.html

```diff
@@ -1,19 +1,275 @@
+          alt="three devices"
+          class="icon__logo"
+        />
+        <p>Internet for all devices</p>
+      </div>
+      <div class="icon">
+        <img src="img/icon-coffee.svg" alt="coffee cup" class="icon__logo" />
+        <p>Boost your productivity</p>
+      </div>
+      <div class="icon">
+        <img src="img/icon-refill.svg" alt="charger icon" class="icon__logo" />
+        <p>Pay as You Go</p>
+      </div>
+    </section>
+    
+    <section class="features">
+      <div class="rectangle">
+        <p class="quote">
+          Wherever I am, I just don't worry about my connection anymore!
+        </p>
+        <button href="#" class="button_color">Get Karma today</button>
+      </div>
+    </section>
+
+    <footer>
+      <h3>Join us on</h3>
+      <a href="#"><img src="./img/twitter-icon.svg" alt="Twitter Logo" /></a>
+      <a href="#"><img src="./img/facebook-icon.svg" alt="Facebook Logo" /></a>
+      <a href="#"
+        ><img src="./img/instagram-icon.svg" alt="Instagram Logo"
+      /></a>
+      <p>© Karma Mobility Inc</p>
+    </footer>
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
+  </body>
+</html>
+
+
+<!-- Store page markup -->
+
+
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <title>Karma</title>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <header class="header">
+      <img src="img/karma-logo.svg" alt="Karma logo" class="header__logo" />
+      <div>
+        <nav>
+          <ul class="navigation__list">
+            <li>
+              <a class="navigation__link__karma" href="index.html"
+                >Meet Karma</a
+              >
+            </li>
+            <li>
+              <a href="#">How It Works</a>
+            </li>
+            <li>
+              <a class="navigation_link" href="#">Store</a>
+            </li>
+            <li>
+              <a href="#">Blog</a>
+            </li>
+            <li>
+              <a href="#">Help</a>
+            </li>
+            <li>
+              <a href="#">Login</a>
+            </li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+    <store_main class="order_section">
+      <div class="form_image">
+        <div class="image">
+          <img
+            src="img/Girl working at desk.png"
+            alt="girl by the desk using computer"
+          />
+        </div>
+
+        <div class="form">
```

Comment:
I think it would be a good idea to group all the inputs and the submit button into a form tag. This way, when the user clicks the submit button, all the values from the inputs in the form will be submitted. Also, the browser will do the validation (like required, minlength, etc.) that you've added.

Reply:
I made some changes, hope it's ok now.

Reply:
I think it's almost there!.. One more thing - you could try moving the submit button inside the form tag as well. Then when you click the button, it will perform the validation and submit the values from the input fields.

Reply:
I put the 'submit' button inside the <form>, I was wondering why there was no validation when I was trying to submit the form without filling it out, now I understand I was missing the 'submit' inside the <form>, thank you for your help.

Reply:
Great!